wit/raw-display.wit

```diff
@@ -0,0 +1,37 @@
+package wasi:webgpu;
```

Comment:
How's this different from frame-buffer.wit?

Reply:
This is a *display API*, frame-buffer is a *graphics API*. But you're right, the intended use is to connect a display to a frame-buffer to draw into, primarily for embedded devices. The difference is that frame-buffer is an arbitrary framebuffer, but this maps to hardware displays and their framebuffers. This is just a proof that multiple display APIs are supported, we don't have to actually include it.

Reply:
Any reason we can't use frame-buffer for both?

Reply:
frame-buffer has no way to specify a screen, or which screen.

Reply:
I see. 
But I'd much rather have one interface for both.
Wanna open a separate issue or PR to discuss?